src/behaviors/loading/loading.js

```diff
@@ -0,0 +1,28 @@
+(function() {
+'use strict';
+
+angular
+  .module('semantic.ui.behaviors.loading', [])
+
+  .directive('smLoading', smLoading);
+
+  smLoading.$inject = ['$parse'];
+
+  function smLoading($parse) {
+    return {
+      restrict:'A',
+      link: function(scope, element, attrs) {
+        scope.$watch(function() {
+          return $parse(attrs.smLoading)(scope);
+        }, function(loading) {
+          if (loading) {
+            element.addClass('loading');
```

Comment:
Using this argument, you could say that there's no point in having an `smLoading` directive at all. I think what we want is a loading directive which is very simple, and very fast, not blocked on ngClass's animation sync stuff unless it's ever needed.